src/Symfony/Bundle/WebProfilerBundle/Controller/ExceptionController.php

```diff
@@ -20,14 +20,14 @@
 use Twig\Error\LoaderError;
 use Twig\Loader\ExistsLoaderInterface;
 
-@trigger_error(sprintf('The "%s" class is deprecated since Symfony 4.4, use "%s" instead.', ExceptionController::class, ExceptionErrorController::class), E_USER_DEPRECATED);
+@trigger_error(sprintf('The "%s" class is deprecated since Symfony 4.4, use "%s" instead.', ExceptionController::class, ExceptionPanelController::class), E_USER_DEPRECATED);
 
 /**
  * ExceptionController.
  *
  * @author Fabien Potencier <fabien@symfony.com>
  *
- * @deprecated since Symfony 4.4, use the ExceptionErrorController instead.
+ * @deprecated since Symfony 4.4, use the ExceptionPanelController instead.
  * @internal since Symfony 4.4
```

Comment:
please remove the internal as that's irrelevant now

Reply:
removed